fili-core/src/main/java/com/yahoo/bard/webservice/async/jobs/payloads/DefaultJobPayloadBuilder.java

```diff
@@ -31,15 +37,16 @@
         Map<String, String> job = new LinkedHashMap<>();
 
         String ticket = jobRow.getId();
-        Map<String, String> fieldValueMap = jobRow.getRowMap();
 
-        job.put("query", fieldValueMap.get(DefaultJobField.QUERY.getName()));
+        job.put(QUERY.getName(), jobRow.get(QUERY));
         job.put("results", JobPayloadBuilder.getResultsUrl(ticket, uriInfo));
         job.put("syncResults", JobPayloadBuilder.getSyncResultsUrl(ticket, uriInfo));
         job.put("self", JobPayloadBuilder.getSelfUrl(ticket, uriInfo));
-        job.put("status", fieldValueMap.get(DefaultJobField.STATUS.getName()));
-        job.put("jobTicket", fieldValueMap.get(DefaultJobField.JOB_TICKET.getName()));
-        job.put("dateCreated", fieldValueMap.get(DefaultJobField.DATE_CREATED.getName()));
+        job.put(STATUS.getName(), jobRow.get(STATUS));
+        job.put(JOB_TICKET.getName(), jobRow.get(JOB_TICKET));
+        job.put(DATE_CREATED.getName(), jobRow.get(DATE_CREATED));
+        job.put(DATE_UPDATED.getName(), jobRow.get(DATE_UPDATED));
+        job.put(USER_ID.getName(), jobRow.get(USER_ID));
```

Comment:
There's almost always a better way to do something.  That's why I love this review process, I get smarter by watching other people's work.